input/fsh/models.fsh

```diff
@@ -79,6 +79,21 @@ criteria are defined by FHIRPath expressions.
   * description 0..1 markdown "Description of the column" """
     A human-readable description of the column.
   """
+  * collection 0..1 boolean "Indicates whether the column may have multiple values." """
+  Indicates whether the column may have multiple values. Defaults to `false` if unset.
+  
+  ViewDefinitions must have this set to `true` if multiple values may be returned. Implementations should
+  report an error if multiple values are produced when that is not the case.
```

Comment:
I think this ought to be SHALL. Or at least implementations SHALL NOT silently exclude values from the output.

Reply:
Done.